ci/package_list.sh

```diff
@@ -7,6 +7,7 @@ RELEASE_PACKAGES=(
     tedge-agent
     tedge-watchdog
     tedge-apt-plugin
+    tedge-log-plugin
     c8y-log-plugin
```

Comment:
So we will build `c8y-log-plugin` as well.

Reply:
Yes at this stage it does us no harm in building the c8y-log-plugin. Then a follow up PR will remove all traces of the c8y-log-plugin in a few weeks time.